tests/fuzz/runFuzzTest.sh

```diff
@@ -228,6 +229,14 @@ then
             printf "Uploading corpus files to S3 bucket...\n"
             aws s3 sync ./corpus/${TEST_NAME}/ $CORPUS_UPLOAD_LOC/${TEST_NAME}/
         fi
+
+        # Store generated output files in the S3 bucket.
+        if [ "$ARTIFACT_UPLOAD_LOC" != "none" ]; then
+            printf "Uploading output files to S3 bucket...\n"
+            aws s3 cp ./${TEST_NAME}_output.txt ${ARTIFACT_UPLOAD_LOC}/${TEST_NAME}/output_$(date +%Y-%m-%d).txt
+            aws s3 cp ./${TEST_NAME}_results.txt ${ARTIFACT_UPLOAD_LOC}/${TEST_NAME}/results_$(date +%Y-%m-%d).txt
+            aws s3 cp ./${TEST_NAME} ${ARTIFACT_UPLOAD_LOC}/${TEST_NAME}/${TEST_NAME}_$(date +%Y-%m-%d)
```

Comment:
Is this an executable binary?  Does it have any linking or other dependency that need to be saved ?  

Also, if we're running these tests several times a day, is `%Y-%m-%d` granular enough (won't these also get over-written?)  Did you consider `%s`?

Reply:
I also made changes so that the output files are uploaded only when the fuzz tests fail, assuming we are more interested in the failure case more than the successful ones. Corpus files that are used at the time of failure are also uploaded with timestamp to help with debugging